server/RemoteHubClient.ts

```diff
@@ -190,4 +199,29 @@ export default class RemoteHubClient extends BaseRemoteClient {
       else callback(null);
     });
   };
+
+  private onImportProject =  (data: ImportProjectData) => {
+
+  fs.writeFile(path.join(this.server.projectsPath, "project.zip"), data.data as string, (err) => {
+    if (err) return console.log(err);
+
+    fs.createReadStream(path.join(this.server.projectsPath, "project.zip"))
+      .pipe(unzip.Parse())
+      .on("entry", (entry: any) => {
+        if (entry.type === "Directory") {
+          fs.mkdir(path.join(this.server.projectsPath, entry.path));
+        }
+        else {
+          entry.pipe(fs.createWriteStream(path.join(this.server.projectsPath, entry.path)));
+        }
+      })
+      .on("close", () => {
+        this.server.loadProject(data.name, (err: Error) => {
+          if (err) {
```

Comment:
We use explicit `if (err != null)` throughout the codebase rather than implicit boolean coercion.